src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -184,6 +184,10 @@ public DescriptorImpl() {
 
         @Override
         public Builder newInstance(StaplerRequest req, JSONObject formData) throws FormException {
+            JSONObject commandMode = (JSONObject) formData.get("commandMode");
```

Comment:
Can this 'formData.get("commandMode")' be null? If so, please check it prior to converting.